packages/taquito-michelson-encoder/src/tokens/pair.ts

```diff
@@ -170,6 +171,28 @@ export class PairToken extends ComparableToken {
     );
   }
 
+  generateSchema(): TokenSchema {
+    return {
+      __michelsonType: PairToken.prim,
+      schema: this.traversal(
```

Comment:
can I have a brief explanation of what this function is doing?

Reply:
Based on a michelson type, "generateSchema" returns the schema as expected by Taquito for the storage or entry point of a contract. 
It can be used to discover how to write the storage property when deploying a contract or the parameter when calling a smart contract entry point using the methodsObject property.

Reply:
thanks for the clarification!